mopidy/__main__.py

```diff
@@ -7,21 +7,23 @@
 import textwrap
 
 try:
-    import gobject   # noqa
+    import gi
```

Comment:
Re the change you did to add a version to the psbutils import, would it make sense to instead just have a `mopidy.audio.internal.gst` module which just does the `gi.require_version` bits plus imports, initialization and we can import from there?